ports/nrf/README.md

```diff
@@ -106,6 +107,7 @@ wt51822_s4at         | s110                    | Peripheral             | Manual
 pca10040             | s132                    | Peripheral and Central | [Segger](#segger-targets)
 feather52            | s132                    | Peripheral and Central | Manual, SWDIO and SWCLK solder points on the bottom side of the board
 arduino_primo        | s132                    | Peripheral and Central | [PyOCD](#pyocdopenocd-targets)
+WRL13990             |                         | Peripheral and Central | [Segger](#segger-targets)
```

Comment:
it would be great to keep the board name in lower case here to keep some kind of style. Also, i'm very confident that Bluetooth LE will run on this board, so you could point it to the s132 like the other nrf52832 targets as that will be the proper SoftDevice for this chip.